src/mgr/PyModules.cc

```diff
@@ -629,3 +631,15 @@ PyObject* PyModules::get_counter_python(
   return f.get();
 }
 
+PyObject *PyModules::get_context()
+{
+  PyThreadState *tstate = PyEval_SaveThread();
+  Mutex::Locker l(lock);
+  PyEval_RestoreThread(tstate);
+
+  // Construct a capsule containing ceph context
+  // TODO: a put() on context as a destructor argument?  and get() here?
```

Comment:
guess it's fine as it is, because the plugin won't outlive the ceph_mgr process, and the life cycle of `g_ceph_context` is managed by the smart pointer returned by `global_init()` which is called in `main()`.